File: java64/nivel1/exer3.java
import java.util.Scanner;

public class exer3 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double pagamento;
        double precoUnitario;
        int idVendedor;
        int qtdVendida;
        int codigoDaPeca;

        System.out.println("ID do vendedor: ");
        idVendedor = input.nextInt();

        System.out.println("Código da peça: ");
        codigoDaPeca = input.nextInt();

        System.out.println("Preço unitário: ");
        precoUnitario = input.nextDouble();

        System.out.println("Quantidade vendida: ");
        qtdVendida = input.nextInt();

        pagamento = (qtdVendida * precoUnitario)  * 0.05;

        System.out.println("ID: " + idVendedor +
        "\nCódigo do produto: " + codigoDaPeca +
        "\nPreço unitário: " + precoUnitario +
        "\nQuantidade vendida: " + qtdVendida +
        "\nPagamento do funcionario: " + pagamento);

        input.close();
    }
}

File: pyhton64/nivel1/exer3.py
id_vendedor = str(input('ID do vendedor: '))
codigo_peca = str(input('Código da peça: '))
preco_unitario = float(input('Preço unitario: '))
quantidade_vendida = int(input('Quantidade vendida: '))

pagamento = float(0.05 * (preco_unitario * quantidade_vendida))
print(f'Id: {id_vendedor}\n' + 
    f'Código da peça: {codigo_peca}\n' + 
    f'Preço unitario: {preco_unitario}\n' + 
    f'Quantidade vendida: {quantidade_vendida}\n' + 
    f'Pagamento: {round(pagamento, 2)}')






